packages/calling/src/CallingClient/line/index.ts

```diff
@@ -263,6 +268,16 @@ export default class Line extends Eventing<LineEventTypes> implements ILine {
 
       return call;
     }
+    if (this.serviceData?.indicator === ServiceIndicator.GUEST_CALLING) {
```

Comment:
Added the 'if' case over here, in case the destination is not present it moves here and then sets up guest calling flow.
Nit: Will need to change the log to include 'guest calling call' probably. Will make the change.